Week-6/Problem 6.3.js

```diff
@@ -0,0 +1,27 @@
+
+console.log("Problem 6.3:-  Sort array of 0's,1's and 2'");
+const arr =[0,0,1,1,1,1,1,2,0,0,0,2];
+console.log(arr);
+let low=0;
+let mid=0;
+let high = arr.length-1;
+while(mid<=high)
+{
+    switch(arr[mid])
+    {
+        
+        case 0:
+               arr[low]=arr[mid];
```

Comment:
mention time and space complexity for each of the problems

Reply:
as per you review comment I have added time and space complexity in each assignment.